webpack.config.js

```diff
@@ -1,35 +1,35 @@
 module.exports = {
   mode: process.env.NODE_ENV ?? "development",
-
+  target: "browserslist:browserslist config, not current node",
```

Comment:
browserlist is default, why need all this?

Reply:
Webpack isn't happy with combining the targets with node. Seems like it's a feature they haven't released. So I excluded node for this build.😕

Reply:
Actually, does node need be targeted too in this config for this lib? I just did a quick test with the umd on my local site and it seems to work fine without it. But if it has to be included, I think there is a workaround to adding it too.

Reply:
no need to target node, it's a react component

Reply:
Removed node target in https://github.com/Open-Attestation/decentralized-renderer-react-components/pull/62/commits/3aa324676f7a3de673bfccc387f44a78bf6204a7